consensus/polybft/state.go

```diff
@@ -256,6 +318,79 @@ func (s *State) list() ([]*StateSyncEvent, error) {
 	return events, nil
 }
 
+// insertStateSyncEvent inserts a new state sync event to state event bucket in db
+func (s *State) insertExitEvent(event *ExitEvent) error {
+	return s.db.Update(func(tx *bolt.Tx) error {
+		raw, err := json.Marshal(event)
+		if err != nil {
+			return err
+		}
+
+		bucket := tx.Bucket(exitEventsBucket)
+
+		return bucket.Put(bytes.Join([][]byte{itob(event.Epoch), itob(event.ID), itob(event.BlockNumber)}, nil), raw)
```

Comment:
to be able to find events by block number i think it worth to have event.BlockNumber first

Reply:
epoch, block number, event Number.

Reply:
I meant switch event number and block number

Reply:
But i realized that it's not important here